deploy/aws/cloudbeat-aws.yml

```diff
@@ -47,7 +47,7 @@ output.elasticsearch:
 
 # ================================= Processors =================================
 processors:
-  - add_host_metadata: ~
+  # - add_host_metadata: ~
```

Comment:
for CSPM let's remove it completely, no? Does it make sense to re-enable it here?